wazimap_np/templates/about.html

```diff
@@ -3,22 +3,22 @@
 <p>NepalMap is a project of <a href="http://codefornepal.org/en/"><strong>Code for Nepal</strong></a>, a non-profit that works to increase digital literacy and the use of open data in Nepal. Please make a </span><a href="https://www.paypal.com/fundraiser/charity/117735" target="_blank"><strong>tax deductible donation</strong></a> to help us maintain the server space for NepalMap, and empower Nepal digitally!</p>
-<p>Code for Nepal's co-founder, <a href="http://codefornepal.org/en/staff/ravinepal/" target="_blank">Ravi Kumar</a>, is grateful to all the volunteers who contributed to NepalMap! Code for Nepal is especially grateful to <a href="https://github.com/cliftonmcintosh" target="_blank">Clifton McIntosh</span></a>, for serving as the lead developer for this project. And to Code for Nepal&rsquo;s </span><a href="http://codefornepal.org/en/staff/amit-chaudhary/" target="_blank">Amit Chaudhary</a>.</p>
+<p>Code for Nepal's founder, <a href="http://codefornepal.org/en/staff/ravinepal/" target="_blank">Ravi Kumar</a>, is grateful to all the volunteers who contributed to NepalMap! Code for Nepal is especially grateful to <a href="https://github.com/cliftonmcintosh" target="_blank">Clifton McIntosh</span></a>, for serving as the lead developer for this project. And to many amazing </span><a href="https://github.com/Code4Nepal/nepalmap_federal/graphs/contributors/" target="_blank">volunteers who made NepalMap possible</a>.</p>
 </p>
 <h2 class="column-full subhead">Shoulders of Giants</h2>
 <p>Yes! NepalMap is based on <a href="https://github.com/Code4SA/wazimap" target="_blank">Wazimap</a> which is a fork of <a href="https://censusreporter.org/" target="_blank">Census Reporter</a>. Wazimap is built and maintained by <a href="http://code4sa.org/" target="_blank">Code for South Africa</a>. Other implementations include <a href="https://kenya.wazimap.org/" target="_blank">Wazimap Kenya</a> and <a href="https://wazimap.co.za" target="_blank">Wazimap South Africa</a>, the first Wazimap implementation. All of these tools are completely open source. You can build <a href="https://wazimap.readthedocs.org/en/latest/index.html" target="_blank">one</a> for your country. NepalMap code is licensed under the MIT License.<p>
 
 <p>NepalMap branding is copyright of Code for Nepal and cannot be used without permission.</p> 
 
 <h2 class="column-full subhead">Help us</h2>
-<p>We are looking for volunteers who can help us add more data, and improve NepalMap. If you have the right skills, and would like to help, please complete this <a href="http://codefornepal.org/en/help/" target="_blank">form</a>! 
+<p>We are looking for volunteers who can help us add more data, and improve Beta version of NepalMap. If you have the right skills, and would like to help, please contribute to NepalMap on <a href="https://github.com/Code4Nepal/nepalmap_federal/issues" target="_blank">GitHub</a>! 
```

Comment:
I would change "improve Beta version" to "improve the Beta version" or "improve the beta version".